src/service.h

```diff
@@ -0,0 +1,20 @@
+#ifndef SERVICE_H
```

Comment:
er is een file #include <tlspool/internal.h> voor dit doel, verplaats dit daar gewoon heen

Reply:
deze file vervalt